core/src/test/java/org/arquillian/smart/testing/report/SmartTestExecutionReportUsingPropertyTest.java

```diff
@@ -0,0 +1,102 @@
+package org.arquillian.smart.testing.report;
+
+import java.io.File;
+import net.jcip.annotations.NotThreadSafe;
+import org.arquillian.smart.testing.Configuration;
+import org.arquillian.smart.testing.TestSelection;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.RestoreSystemProperties;
+import org.junit.experimental.categories.Category;
+
+import static java.util.Arrays.asList;
+import static org.arquillian.smart.testing.Configuration.SMART_TESTING;
+import static org.arquillian.smart.testing.report.SmartTestExecutionReport.SMART_TESTING_REPORT_DIR;
+import static org.arquillian.smart.testing.report.SmartTestExecutionReport.SMART_TESTING_REPORT_NAME;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Category(NotThreadSafe.class)
```

Comment:
we do have default values for report directory which is I mention in documentation. Here we are verifying, if we set dir other than default, is it behaving as expected?